doc/rados/operations/balancer.rst

```diff
@@ -135,13 +139,40 @@ There are two supported balancer modes:
 
    To use ``upmap``, all clients must be Luminous or newer.
 
-The default mode is ``upmap``. The mode can be changed to ``crush-compat`` by
-running the following command:
+#. **read**. In Reef and later clients, the OSDMap can store explicit
```

Comment:
Do we mean "clients" here?  Or maybe "Ceph releases beginning with Reef" ?  And update the below sentence about client compatibility to note a minimum kernel version for KRBD?

Reply:
I meant to say "releases" instead of "clients". Fixed!

Not sure about a minimum kernel version for KRBD- @JoshSalomon is this something you would know?

Reply:
Nope  :-(